lib/resolvers/project-version.js

```diff
@@ -45,7 +45,7 @@ module.exports = ({ models }) => ({ action, data, id }, transaction) => {
         } else if (conditions) {
           versionData.conditions = conditions;
         }
-        if (retrospectiveAssessment) {
+        if (!isUndefined(retrospectiveAssessment)) {
           versionData.retrospectiveAssessment = retrospectiveAssessment;
```

Comment:
Let's make it actually a `Boolean`
```suggestion
          versionData.retrospectiveAssessment = !!retrospectiveAssessment;
```